src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -449,16 +499,28 @@ public int getTotalUnregisteredInCourse(String courseId) throws EntityDoesNotExi
         return studentsLogic.getUnregisteredStudentsForCourse(courseId).size();
     }
 
-    public CourseDetailsBundle getCourseSummary(CourseAttributes cd) throws EntityDoesNotExistException {
-        Assumption.assertNotNull("Supplied parameter was null\n", cd);
+    /**
+     * Gets the {@link CourseDetailsBundle} for a course using {@link CourseAttributes}.
```

Comment:
Done. Edited all descriptions. Did not include attributeName in the `@return` tags. For an example, I think it makes more sense to say,
`@return a list of {@link CourseDetailsBundle}`
thank
`@return a list of {@link CourseDetailsBundle courseDetails}`

Tell me what you think.